### DISCUSSION TITLE
signac and scATACseq data

### DISCUSSION THREAD

**Discussion by SuqinYang** [stuart-lab/signac#1176] [Q&A] [answered]
Hi,sorry to trouble you.I want to use signac to analyze the scATACseq data I downloaded from the database.However,the sequencing data for each cell is a pair of separate fastq files. It's not from 10x platform，so how to get  the Peak/Cell matrix  and Fragment file in the tutorial “https://satijalab.org/signac/articles/pbmc_vignette.html” ?
How do I use signac in this data?

**Comment by timoast** [comment, marked as answer]
You can create a fragment file from a BAM file using the `sinto` package, see documentation here: https://timoast.github.io/sinto/basic_usage.html#create-scatac-seq-fragments-file

If you have reads from each cell existing as a separate file, you could add some barcode information to each file and then combine them and map the data. We don't have tutorials showing how to do this, since it's quite specific to individual datasets.
